install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1960,6 +1960,13 @@ hubble:
       # - source
       # - destination
       # - verdict
+      fieldAggregate: []
+      # - time
+      # - source
+      # - destination
+      # - verdict
+      aggregationInterval: "0s"
+      # --- Defines the interval at which to aggregate before exporting Hubble flows.
```

Comment:
This comment seems to be on the wrong line. Also please mention that 0s means that aggregation is disabled.